patches/devel.R-cran-whisker.0.4.diff

```diff
@@ -0,0 +1,54 @@
+diff --git a/devel/Makefile b/devel/Makefile
+index 6ec52280ce90..7378354638aa 100644
+--- a/devel/Makefile
++++ b/devel/Makefile
+@@ -105,6 +105,7 @@
+     SUBDIR += R-cran-uuid
+     SUBDIR += R-cran-vcd
+     SUBDIR += R-cran-vctrs
++    SUBDIR += R-cran-whisker
+     SUBDIR += R-cran-withr
+     SUBDIR += R-cran-zeallot
+     SUBDIR += RStudio
+diff --git a/devel/R-cran-whisker/Makefile b/devel/R-cran-whisker/Makefile
+new file mode 100644
+index 000000000000..dfb56539d6af
+--- /dev/null
++++ b/devel/R-cran-whisker/Makefile
+@@ -0,0 +1,18 @@
++# Created by: Guangyuan Yang <ygy@FreeBSD.org>
++# $FreeBSD$
++
++PORTNAME=	whisker
++DISTVERSION=	0.4
++CATEGORIES=	devel
++DISTNAME=	${PORTNAME}_${DISTVERSION}
++
++MAINTAINER=	ygy@FreeBSD.org
++COMMENT=	{{mustache}} for R, Logicless Templating
```

Comment:
`{{mustache}}` looks a bit weird, is that intentional?

Reply:
Yup that is how the mustache template looks like haha! I could change it to normal if you feel like this is too much XD

Reply:
I'm fine If it shows fine in pkg's metadata.

Reply:
I will actually change it to `Logicless mustache templating for R` just to be safe.